styles/styles.css

```diff
@@ -150,9 +146,12 @@ h1, h2 {
 
 h3 {
   display: block;
-  font-size: 2.8rem;
-  line-height: 3.64rem;
+  font-size: var(--heading-font-size-l);
```

Comment:
I think we should be consistent. 
Either
1. (best practice) we assign a font-size variable on each header type (e.g. h3 `L` , h1 `XL`) 
and then we adjust these variables with media queries (example from the boilerplate https://github.com/adobe/aem-boilerplate/blob/main/styles/styles.css#L53-L62)
2. we have font-sizes everywhere and not use variables

But we should not mix them, because otherwise it gets confusing.
I indeed was rushing when I made the initial stylings for the demo for the initial kick off.

Reply:
fixed in acb55e0